x11-drivers/nvidia-drivers/files/bin/setup-nvidia

```diff
@@ -3,9 +3,8 @@ set -euo pipefail
 
 . /usr/share/flatcar/release
 . /usr/share/flatcar/update.conf
+. /usr/share/nvidia/release
 
-NVIDIA_DRIVER_VERSION=450.80.02
-NVIDIA_PRODUCT_TYPE=tesla
```

Comment:
Can the product type change with the release version? I thought it was associated with the card in the machine and thus it would be hardware dependent, not release dependent, no?

Reply:
> Can the product type change with the release version? I thought it was associated with the card in the machine and thus it would be hardware dependent, not release dependent, no?

No, the product type cannot change with the release version. `tesla` is the name behind the data center GPUs that NVIDIA manufactures. The product type is an umbrella for a couple of hardware in that category. Though I read somewhere NVIDIA is planning to drop the name `tesla` to avoid confusion.